backend/dataall/modules/datapipelines/db/datapipelines_repositories.py

```diff
@@ -150,7 +150,7 @@ def query_pipeline_environments(session, uri) -> Query:
         query = session.query(DataPipelineEnvironment).filter(
             DataPipelineEnvironment.pipelineUri.ilike(uri + '%%'),
         )
-        return query
+        return query.order_by(DataPipelineEnvironment.envPipelineUri)
```

Comment:
if the label is available maybe better to order by label or name

Reply:
rather than label - do you mean by `stage` name (i.e. dev, test, prod, etc.)? I can make that change

Reply:
switched